src/Compilers/Core/Portable/Emit/SemanticEdit.cs

```diff
@@ -72,7 +77,7 @@ public struct SemanticEdit : IEquatable<SemanticEdit>
         /// <exception cref="ArgumentOutOfRangeException">
         /// <paramref name="kind"/> is not a valid kind.
         /// </exception>
-        public SemanticEdit(SemanticEditKind kind, ISymbol? oldSymbol, ISymbol? newSymbol, Func<SyntaxNode, SyntaxNode?>? syntaxMap = null, bool preserveLocalVariables = false)
+        public SemanticEdit(SemanticEditKind kind, ISymbol? oldSymbol, ISymbol? newSymbol, Func<SyntaxNode, SyntaxNode?>? syntaxMap = null, bool preserveLocalVariables = false, SemanticEditOption options = SemanticEditOption.None)
```

Comment:
>SemanticEdit [](http://example.com/codeflow?start=15&length=12)

Do we need to add an overload with the previous signature, for backward compatibility?

Reply:
Thanks, added.